### PR TITLE
Fix wso2/product-apim#6127

### PR DESCRIPTION
**Pull request by vithu30**
Fix https://github.com/wso2/product-apim/issues/6127

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx [hunk 1]
```diff
@@ -185,7 +185,11 @@ function EditMediationPolicy(props) {
             })
             .catch((errorResponse) => {
                 console.log(errorResponse);
-                Alert.error(JSON.stringify(errorResponse));
+                if (errorResponse.response.body.description !== null) {
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx]: let's add the i18n support
intl.formatMessage(

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx [hunk 2]
```diff
@@ -185,7 +185,11 @@ function EditMediationPolicy(props) {
...
+                    Alert.error(errorResponse.response.body.description);
+                } else {
+                    Alert.error('Error when adding mediation policy');
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx]: here as well

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx (hunk 1 above)
Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx]: This error message is sent from back-end which can be different for different scenarios. Hence, we could not add i18n support for this.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx (hunk 2 above)
Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/MediationPolicies/EditMediationPolicy.jsx]: Fixed

**Pull request merged**